src/main/java/uk/gov/ons/census/fwmt/jobservice/rest/client/CometRestClient.java

```diff
@@ -98,7 +98,8 @@ private void auth() throws GatewayException {
     }
 
     HttpEntity<?> body = new HttpEntity<>(caseRequest, httpHeaders);
-    restTemplate.exchange(basePathway, HttpMethod.PUT, body, Void.class);
+    ResponseEntity<Void> exchange = restTemplate.exchange(basePathway, HttpMethod.PUT, body, Void.class);
```

Comment:
Does this also capture a `body`, or does treating it as `Void` remove that? It is possible that important information could be contained in the `body`.